tide_data_driven_component.module

```diff
@@ -25,10 +25,9 @@ use Drupal\field\Entity\FieldConfig;
  * @see options_allowed_values()
  */
 function tide_data_driven_component_items_list(FieldStorageConfig $definition, ContentEntityInterface $entity = NULL, $cacheable) {
-  $options = [
-    'myvic_vicfreewifi' => 'Vic Free Wifi Map',
-    'myvic_family_violence_women_programs' => 'Prevention of Family Violence Against Women Programs Map',
-  ];
+  if (!$opts_env = getenv('DATA_DRIVEN_OPTIONS'))
+    $opts_env = '{"myvic_vicfreewifi":"Vic Free Wifi Map","myvic_family_violence_women_programs":"Prevention of Family Violence Against Women Programs Map"}';
+  $options = json_decode($opts_env, TRUE);
```

Comment:
Is this really necessary? i.e. creating a JSON object and then decoding it to get a PHP array? Just set it to a PHP array.

Reply:
oh wait, there's no braces for the if statement. Did lint let you get away with that?